repology/parser/slackware.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (C) 2016 Dmitry Marakasov <amdmi3@amdmi3.ru>
+# Copyright (C) 2017 David Spencer <idlemoor@slackbuilds.org>
+#
+# This file is part of repology
+#
+# repology is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# repology is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with repology.  If not, see <http://www.gnu.org/licenses/>.
+
+import re
+import sys
+
+from repology.package import Package
+
+
+class SlackwareParser():
+    def __init__(self):
+        pass
+
+# pylint: disable=R0201
+    def Parse(self, path):
+        packages = {}
+
+        # There is only The Man :)
+        volkerdi = ['volkerdi@slackware.com']
+
+        # Look for packages in these top level directories:
+        pkgdirs = ['slackware', 'slackware64', 'slackwarearm', 'patches', 'extra']
```

Comment:
This was in preparation for adding the 64 bit and ARM repos, I have now added them to repos.d/slackware.yaml (32 bit and 64 bit are now subrepos, but SlackwareARM is separate because the upstream URLs are different).